source/array_table_model.ts

```diff
@@ -9,25 +10,38 @@ export class ArrayTableModel extends TableModel {
   constructor() {
     super();
     this.dispatcher = new Kola.Dispatcher<Operation>();
+    this.table = [];
+    this.transactionArray = null;
   }
 
   /**
    * Marks the beginning of a transaction. In cases where a transaction is
    * already being processed, then the sub-transaction gets consolidated into
    * the parent transaction.
    */
-  public beginTransaction(): void {}
+  public beginTransaction(): void {
+    if(this.transactionArray === null) {
+      this.transactionArray = [];
+    }
+  }
 
   /** Ends a transaction. */
-  public endTransaction(): void {}
+  public endTransaction(): void {
+    if(this.transactionArray !== null) {
```

Comment:
This won't handle nested transactions properly, you can write a unit test to see why.